sdks/python/apache_beam/io/gcp/bigquery_file_loads_test.py

```diff
@@ -176,7 +176,7 @@ def test_many_files(self):
     file length is very small, so only a couple records fit in each file.
     """
 
-    fn = bqfl.WriteRecordsToFile(schema=_ELEMENTS_SCHEMA, max_file_size=300)
+    fn = bqfl.WriteRecordsToFile(schema=_ELEMENTS_SCHEMA, max_file_size=50)
```

Comment:
Why change these? (Not opposed just curious for the record)

Reply:
50 and 300 are the file sizes that force the transform to spill out after a couple elements in JSON and AVRO respectively.